main.go

```diff
@@ -71,6 +72,33 @@ func main() {
 		}
 	}
 
+	// The goroutine will continuously check bitcoind usage and auto close bitcoind if the client isn't active.
```

Comment:
@lemonlatte I remember we said we would only introduce the new pod design into this cycle, without stopping bitcoind yet. Is this still true?